print/components/PicassoChunk.vue

```diff
@@ -21,14 +21,28 @@
         event-overlap-mode="column"
         first-interval="0"
         interval-count="24"
-        interval-height="35"
+        :interval-height="landscape ? 23 : 35"
         interval-width="46"
         event-text-color="black"
         :locale="$i18n.locale"
         :interval-format="intervalFormat"
         :day-format="dayFormat"
         :weekday-format="weekdayFormat"
       >
+        <!-- day header -->
+        <template #day-label-header="{ date }">
+          <span class="tw-block">
+            {{ $date.utc(date).format($tc('global.datetime.dateLong')) }}
+          </span>
+
+          <span v-if="hasDayResponsibles(date)" class="tw-text-sm tw-italic">
+            {{ $tc('entity.day.fields.dayResponsibles') }}:
+            {{ dayResponsiblesCommaSeparated(date) }}
+          </span>
+
+          <!-- <day-responsibles :date="date" :period="period" :readonly="!editable" /> -->
```

Comment:
Remove?

Reply:
Fixed in https://github.com/ecamp/ecamp3/pull/3026/commits/50484442f0499ead958511a32abb2a1e962c55c4